src/sidebar/components/menu-item.js

```diff
@@ -56,57 +57,74 @@ export default function MenuItem({
   const leftIcon = isSubmenuItem ? null : renderedIcon;
   const rightIcon = isSubmenuItem ? renderedIcon : null;
 
-  return (
-    <Fragment>
-      {/* FIXME-A11Y */}
-      {/* eslint-disable-next-line jsx-a11y/role-supports-aria-props */}
-      <div
-        aria-checked={isSelected}
-        className={classnames('menu-item', {
+  let menuItem = null;
+
+  if (href) {
+    // link item
+    //
+    menuItem = (
```

Comment:
Probably will slice this out into its other component, maybe at the top of this file.